config/environments/test.rb

```diff
@@ -57,4 +57,7 @@
 
   # Enable automatic creation/migration of the test DB when running tests
   config.active_record.maintain_test_schema = true
+
+  # Maintain standard order of running tests in case of leaking changes
+  config.active_support.test_order = :sorted
```

Comment:
I ran a few tests with it set to random and I'm sure they already are order dependent (since :sorted is the current Rails/develop behaviour), and [we're not alone](https://github.com/rails/rails/blob/v4.2.6/activerecord/test/cases/helper.rb#L210).

I think the affected test cases are:

* Api::V2::OperatingsystemsControllerTest
* Api::V2::OsDefaultTemplatesControllerTest
* Api::V2::OverrideValuesControllerTest
* Api::V2::ParametersControllerTest
* Api::V2::PuppetclassesControllerTest
* Api::V2::ReportsControllerTest
* Api::V2::RolesControllerTest
* Api::V2::SmartClassParametersControllerTest
* Api::V2::SmartProxiesControllerTest
* Api::V2::SmartVariablesControllerTest
* Api::V2::SubnetsControllerTest
* Api::V2::TasksControllerTest
* Api::V2::TemplateCombinationsControllerTest
* SeedsTest

Jenkins runs:

* http://ci.theforeman.org/job/test_develop_pr_core/8473/
* http://ci.theforeman.org/job/test_develop_pr_core/8474/
* http://ci.theforeman.org/job/test_develop_pr_core/8475/

I'll take a closer look at each test soon and see if I can address the root causes in develop, then when it's stable we can flip this to random on this branch.  Is that OK?